portfolio/src/main/webapp/index.html

```diff
@@ -11,7 +11,7 @@
     <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/umd/popper.min.js"></script>
     <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.0/js/bootstrap.min.js"></script>
   </head>
-  <body onload="showFact()">
+  <body onload="showFact(); getMessages();">
```

Comment:
How about using a onload function in your script that calls these two functions, and just call onload() here. It might be cleaner as you add more stuff to this line.

Reply:
Done!